test/unit_test/pmf_to_pdf_test.cc

```diff
@@ -115,20 +120,20 @@ BOOST_AUTO_TEST_CASE(pmf_to_pdf_basic)
   float max_val = 1100;
 
   uint32_t action = 2;
-  const VW::pmf_to_pdf::predictions_t prediction_scores{{action, 1.f}};
+  const ::predictions_t prediction_scores{{action, 1.f}};
 
-  const auto test_harness = VW::pmf_to_pdf::get_test_harness_reduction(prediction_scores);
+  const auto test_harness = ::get_test_harness_reduction(prediction_scores);
 
   VW::example ec;
 
-  auto data = scoped_calloc_or_throw<VW::pmf_to_pdf::reduction>();
+  auto data = scoped_calloc_or_throw<VW::reductions::pmf_to_pdf_reduction>();
```

Comment:
is the second `_reduction` redundant here?

Reply:
I think that just `pmf_to_pdf` is ambiguous when in that namespace there will be a `pmf_to_pdf_reduction` and `pmf_to_pdf_setup`